Public/Src/Cache/ContentStore/Distributed/NuCache/ContentLocationDatabase.cs

```diff
@@ -131,6 +170,18 @@ protected override Task<BoolResult> StartupCoreAsync(OperationContext context)
                     Timeout.InfiniteTimeSpan);
             }
 
+            if (_configuration.CacheEnabled && _configuration.CacheFlushingMaximumInterval != Timeout.InfiniteTimeSpan)
+            {
+                _inMemoryCacheFlushTimer = new Timer(
+                    _ => {
+                        Counters[ContentLocationDatabaseCounters.NumberOfCacheFlushesTriggeredByTimer].Increment();
+                        ForceCacheFlush(context);

```

Comment:
>ForceCacheFlush [](start = 24, length = 15)

Maybe just call ForceCacheFlushAsync().FireAndForget() to avoid blocking two threads.